__tests__/app.test.js

```diff
@@ -39,3 +39,37 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("GET 200: sends a single article to the client", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
```

Comment:
If you did {body} here instead of response then below you could do 

body.article.article_id etc. 

If you did {body: {article}} here instead of response then below you could do 

article.article_id 


I hope that explains destructuring a bit better to you!